qdrant-landing/content/blog/qdrant-1.14.x.md

```diff
@@ -0,0 +1,269 @@
+---
+title: "Qdrant 1.14 - Reranking Support & Extensive Resource Optimizations"
+draft: false
+short_description: "Qdrant 1.14 adds Score-Boosting Reranker for custom search ranking and improved performance optimizations."
+description: "Qdrant 1.14 introduces Score-Boosting Reranker for custom search ranking, and improved resource utilization." 
+preview_image: /blog/qdrant-1.14.x/social_preview.png
+social_preview_image: /blog/qdrant-1.14.x/social_preview.png
+date: 2025-04-14T00:00:00-08:00
+author: David Myriel
+featured: true
+tags:
+  - vector search
+  - vector database
+  - semantic search
+  - release notes
+  - performance optimization
+  - reranking
+  - search engine
+  - database
+  - AI
+  - machine learning
+---
+
+[**Qdrant 1.14.0 is out!**](https://github.com/qdrant/qdrant/releases/tag/v1.14.0) Let's look at the main features for this version:
+
+**Score-Boosting Reranker:** Blend vector similarity with custom rules and context.</br>
+**Improved Resource Utilization:** CPU and disk IO optimization for faster processing.</br>
+
+**Incremental HNSW Indexing:** Build indexes gradually as data arrives.</br>
+**Batch Search:** Optimized parallel processing for multiple queries.</br>
+
+**Memory Optimization:** Reduced usage for large datasets with improved ID tracking.</br>
+**RocksDB to Gridstore:** Additional reliance on our custom key-value store. </br>
```

Comment:
We use it because it's more 'optimized' for our use case

```suggestion
**RocksDB to Gridstore:** Additional reliance on our optimized key-value store.</br>
```